### ISSUE TITLE
loss rise after 6k

### ISSUE THREAD

**Issue by tuntun990606** [tuanh123789/AdaSpeech#12]
Hello, I use the AIShell dataset to synthesize very poorly, making it difficult to read the entire sentence and only uttering a few syllables. (However, I previously used the Vivos Vietnamese training set to produce a fairly good result. Although I don't understand Vietnamese very well, it is at least fluent, so I think the code should be okay.) I observed your total_ Loss began to rise after 8K steps, and my model also had similar problems at 6K steps. Besides, my phone_ level_loss has been in a state of shock. Do you know the probable cause?
![image](https://user-images.githubusercontent.com/118953161/226503370-2d83cc40-474a-42d9-81e9-1f6ba48ca4df.png)
![image](https://user-images.githubusercontent.com/118953161/226503398-4adeb494-5e95-4032-b4a7-4db32273189e.png)

**Comment by tuanh123789** [comment]
in train config "phoneme_level_encoder_step=60000". It mean before 60000 steps, no gradient apply for "phoneme_level_predictor" and i do not add "phone_level_loss" in "total_loss", after 60000 step gradient is apply for "phoneme_level_predictor" and "phone_level_loss" is add to "total_loss" ( Hence the toal loss rise from 60000)

**Comment by tuanh123789** [comment]
You can check the loss function here
![image](https://user-images.githubusercontent.com/52446297/226504523-31d843eb-e48f-4aff-9903-c3c9e770813a.png)

**Comment by tuntun990606** [comment]
> You can check the loss function here ![image](https://user-images.githubusercontent.com/52446297/226504523-31d843eb-e48f-4aff-9903-c3c9e770813a.png)

thank you , i will check it
